Makefile

```diff
@@ -1,6 +1,6 @@
 GOFMT ?= gofmt -s
 GOFILES := $(shell find . -name "*.go" -type f -not -path "./vendor/*")
-RELEASE_OPTIONS := -ldflags "-s -w" -tags release
+RELEASE_OPTIONS := -ldflags "-s -w -X github.com/goby-lang/goby/vm.DefaultLibPath=${GOBY_LIBPATH}"
```

Comment:
I think we should keep the release tag, because that prevents some testing purpose helper functions get compiled.

Reply:
Definitely. That's a mistake :blush: